pint_pandas/testsuite/test_pandas_extensiontests.py

```diff
@@ -314,74 +313,152 @@ def test_groupby_extension_no_sort(self, data_for_grouping):
 
         index = pd.Index._with_infer(index, name="B")
         expected = pd.Series([1.0, 3.0, 4.0], index=index, name="A")
-        self.assert_series_equal(result, expected)
+        tm.assert_series_equal(result, expected)
 
 
-class TestInterface(base.BaseInterfaceTests):
```

Comment:
were these meant to be removed?

Reply:
I pulled it out because I didn't think it tested anything.  I put it back.